tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html

```diff
@@ -57,6 +59,31 @@ <h3>Profiling isn’t supported in Colab yet.</h3>
     </p>
   </div>
 </template>
+<paper-toast id="toast" text="" always-on-top></paper-toast>
+<paper-dialog id="captureProfileDialog" modal>
+  <paper-input label="Profiler Service URL or TPU name"
+               always-float-label
+               placeholder="localhost:6009"
+               value="{{_profilerServiceAddress}}"></paper-input>
+  <label id="profiler-address-type-label">Address Type:</label>
+  <paper-radio-group selected="ip-address"
```

Comment:
sorry again, it looks like you have duplicate property `selected` here. I believe this one can be omitted.

Reply:
Removed.